tests/meta/delivery_report/conftest.py

```diff
@@ -0,0 +1,41 @@
+"""
+
+
+@pytest.fixture
+def balsamic_tga_analysis():
+    return BalsamicAnalysis(
+        balsamic_config=create_autospec(
+            BalsamicConfigJSON,
+            analysis=create_autospec(BalsamicConfigAnalysis, sequencing_type="targeted"),
+            panel=create_autospec(
+                BalsamicConfigPanel,
+                capture_kit="my_fav_capture_kit",
+                capture_kit_version="best_version",
+            ),
+        ),
+        sample_metrics={
+            "sample_id": BalsamicTargetedQCMetrics(
+                percent_duplication=0.01,
+                compare_predicted_to_given_sex="male",
+                mean_insert_size=1.0,
+                fold_80_base_penalty=1.0,
+                gc_dropout=1.0,
+                median_target_coverage=1.0,
+                pct_target_bases_250x=0.01,
+                pct_target_bases_500x=0.01,
```

Comment:
These values need to match the `EXPECTED_BALSAMIC_QC_TABLE` defined above. Should we link them using variables and format the string?